fireapp/lib/presentation/volunteer_information/volunteer_information.dart

```diff
@@ -0,0 +1,512 @@
+                            .of(context)
+                            .textTheme
+                            .bodyMedium,
+                        textAlign: TextAlign.left,
+                      ),
+                      Expanded(
+                        child: Text(
+                          (AppLocalizations.of(context)?.volunteer_possibleRoles
+                            (volunteerInformation.possibleRoles)
+                              ?? 'Volunteer Possible Roles')
+                              .toString()
+                              .replaceAll('[', '')
+                              .replaceAll(',', '\n')
+                              .replaceAll(']', ''),
+
+                          style: Theme
+                              .of(context)
+                              .textTheme
+                              .bodyMedium,
+                          textAlign: TextAlign.right,
+                        ),
+                      ),
+
+                    ],
+                  ),
+                ),
+                Padding(
+                  padding: EdgeInsets.only(bottom: 16.0),
+                  child:Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Text(
+                        'Volunteer Qualifications:',
+                        style: Theme
+                            .of(context)
+                            .textTheme
+                            .bodyMedium,
+                        textAlign: TextAlign.left,
+                      ),
+                      Expanded(
+                        child: Text(
+                          AppLocalizations.of(context)?.volunteer_qualifications(
+                              volunteerInformation.qualifications.map((q) =>
+                              q.name
+                              )
+                                  .toList()
+                                  .toString()
+                                  .replaceAll('[', '')
+                                  .replaceAll(',', '\n')
+                                  .replaceAll(']', '')
+                          ) ?? 'Volunteer Qualifications',
+                          style: Theme
+                              .of(context)
+                              .textTheme
+                              .bodyMedium,
+                          textAlign: TextAlign.right,
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    Text(
+                      'Monday:',
+                      style: Theme
+                          .of(context)
+                          .textTheme
+                          .bodyMedium,
+                      textAlign: TextAlign.left,
+                    ),
+                    Expanded(
```

Comment:
Make this a widget. You've copy pasted it 7 times, which means it should probably just be written once and reused as a widget.